pkg/test/component_sns_sqs.go

```diff
@@ -2,131 +2,95 @@ package test
 
 import (
 	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
 	"github.com/aws/aws-sdk-go/service/sns"
 	"github.com/aws/aws-sdk-go/service/sqs"
-	"log"
-	"net"
 	"strings"
-	"time"
 )
 
-type snsSqsConfig struct {
-	Debug   bool   `mapstructure:"debug"`
-	Host    string `mapstructure:"host"`
-	SnsPort int    `mapstructure:"sns_port"`
-	SqsPort int    `mapstructure:"sqs_port"`
+type SnsSqsTestClients struct {
+	Sns *sns.SNS
+	Sqs *sqs.SQS
 }
```

Comment:
Please also remove the OnDestroy func() in docker.go

Reply:
✅